time stamp
# receiving time

uint16 numSV             
# Number of SVs used in Nav Solution

uint16 fixType           
# GNSS fix Type, range 0..5

float64 pDOP             
# Position DOP [1 / 0.01]

float64 latitude
float64 longitude
float64 altitude
# Gnss localization (deg/deg/meter)

float64 yaw
float64 pitch
float64 roll
# Gnss orientation (deg)